assets/data/effects/drauven/drauven_pc_flybyattack.json

```diff
@@ -0,0 +1,88 @@
+{
+"id": "drauven_pc_flybyattack",
+"info": {
+	"dataVersion": 1,
+	"sourceFile": "drauven/drauven_pc_flybyattack",
+	"author": "Bipen",
+	"tags": [ "okToMarkDone", "sneaky", "melee" ]
+},
+"type": "ABILITY",
+"verb": "MANEUVER",
+"ability": {
+	"icon": "fly",
+	"priority": "1",
+	"showInCharacterSheet": false,
+	"cooldown": "everyOtherTurn"
+},
+"targets": [
+	{
+		"template": "SELF",
+		"STUB": "Can't be pinned for the base ability, can be for the upgrade",
+		"choose": "BY_SCORE",
+		"scoreFunction": "(1-min(1,pinned))+warriorDeck_zealousLeap_upgrade",
+		"scoreThreshold": "1",
+		"cost": { "class": "ActionPoints", "actionPoints": "free" },
+		"missionFeedback": { "orientTo": "targetTile" },
+		"aspectValues": [
+			{ "id": "drauven_pc" }
+		]
+	},
+	{
+		"role": "targetTile",
+		"template": "TILE",
+		"missionFeedback": {
+			"rangeFeedback": "FRIENDLY_EFFECT_RANGE",
+			"suggestionFeedback": "FRIENDLY_EFFECT_SUGGESTION_SHINE",
+			"hoverFeedback": "PATH_TARGET",
+			"applyRoleFootprint": "self",
+			"showMovementFeedback": true
+		},
+		"minRange": "0.9",
+		"range": "6.6",
+		"lineOfSight": "TERRAIN_ONLY",
+		"tileFilter": "validMoveEnd",
+		"roleMustFit": "self"
+	},
+	{
+		"template": "ADJACENT_ENEMY",
+		"choose": "ANY",
+		"aspects": [ "HOSTILE_TARGET", "alive" ],
+		"range": "1.6",
+		"relativeToLocation": "targetTile"
+	}
+],
+"outcomes": [
+	{ "class": "Animate" }
+	{
+		"class": "MoveEntity",
+		"directlyToTile": "targetTile",
+		"animationType": "leap"
+	},
+	{
+		"class": "AttackRoll",
+		"defenseRollTags": [ "melee" ],
+		"attackStrength": "self.MELEE_ACCURACY",
+		"attackAnimationType": "melee",
+		"particles": [
+			{ "script": "vultureClaws", "setPosition": "targetTile", "delay": -0.25 }
+		],
+		"audioOverride": {
+			"attackHitCue": "Characters/Heroes/Themed/attackClaw",
+			"attackDodgedCue": "Characters/Weapons/Dagger/weaponDodge",
+			"attackBlockedCue": "Characters/Weapons/Dagger/weaponBlock"
+		},
+		"attackerAnimationOverride": "",
+		"cameraSnapToStartPositionOnEnd": false,
+		"always": {
+			"class": "DoAll",
+			"outcomes": [
+				{
+					"class": "Damage",
+					"target": "target",
+					"amount": "4+(self.PHYSICAL_DAMAGE_BONUS+(self.POTENCY*2))",
```

Comment:
This seems like very high damage for this move, especially if we're also airlifting an ally at the same time